Expensify/App
# Remove default state AK and require users to enter a state

@pecanoro, please review when you get the chance

### Details
Changes the default state in the selection tool to be `''` (which will show up as `-`) instead of `AK` which users might forget to update and put the wrong address in for. Also added growl errors if the empty state selections (exception of incorporationState, [since that'll have its own error from the API](https://github.com/Expensify/Expensify.cash/blob/78164949343c427101b4e399c7f233f72ffb19ea/src/libs/actions/BankAccounts.js#L701-L702)). This way they'll have to consciously select a state in every selector. There are appropriate 

### Fixed Issues
$ https://github.com/Expensify/Expensify/issues/170025

### Tests
Did the QA steps locally on web (This is the only platform where we can access bank account setup right now).

### QA Steps
1. Go to staging.expensify.cash/bank-account/new and go through the setup for a new bank account
2. During the setup in the Company Step, the Requestor Step, and the Benefactors step, verify that the `State` selectors show up as `-` by default. Make sure that for each page, it won't let you continue until you choose a valid state (even for Incorporation State). See screenshots below for reference for if anything looks off.
3. While you're completing this, be sure to jump back steps sometimes and verify that the information is as you entered. Specifically, make sure the State isn't being reset to `AK` and is what you selected.

### Tested On


### Screenshots


#### Web
![image](https://user-images.githubusercontent.com/19364431/125690959-c1b247e5-c975-4b05-aa01-cfc97259b242.png)
![image](https://user-images.githubusercontent.com/19364431/125682283-aca8a96c-1051-4a62-9fe0-16bfca23e18b.png)
![image](https://user-images.githubusercontent.com/19364431/125691075-5bcd9501-e79e-45dc-b034-5212c6859df1.png)
![image](https://user-images.githubusercontent.com/19364431/125691142-be3114f9-8926-45f4-ade6-e284b582ed1c.png)

#### Mobile Web


#### Desktop


#### iOS


#### Android
